ai_services/anomaly_detection/data_preprocessing_examples/oci_data_flow_based_examples/requirements.txt

```diff
@@ -2,3 +2,10 @@
 /opt/dataflow/prepackaged_df-1.0-py3-none-any.whl
 oci==2.74.0
 pandas==1.1.5
+
+setuptools~=49.6.0
```

Comment:
what does ~= do?

Reply:
Sure, `~=` is used when any non-breaking and updated version of the specified version can be used.